clang/test/Analysis/out-of-bounds-diagnostics.c

```diff
@@ -278,6 +278,23 @@ int *mallocRegion(void) {
   return mem;
 }
 
+int *custom_calloc(size_t a, size_t b) {
+  size_t res;
+  if (__builtin_mul_overflow(a, b, &res))
+    return 0;
+
+  return malloc(res);
+}
+
+int *mallocRegionOverflow(void) {
+  int *mem = (int*)custom_calloc(4, 10);
```

Comment:
```suggestion
  int *mem = (int*)custom_calloc(10, sizeof(int));
```
Let's pass the nmemb, and then size, as in calloc :)